service/service.go

```diff
@@ -221,7 +221,7 @@ func New(config Config) (*Service, error) {
 		}
 
 		retryWrapConfig := retryresource.DefaultWrapConfig()
-		retryWrapConfig.BackOffFactory = func() backoff.BackOff { return backoff.NewExponentialBackOff() }
+		retryWrapConfig.BackOffFactory = func() backoff.BackOff { return backoff.WithMaxTries(backoff.NewExponentialBackOff(), 3) }
```

Comment:
This is nice I didn't know that function. I think `3` should be a const.

Reply:
It is quite new. See https://github.com/giantswarm/kvm-operator/pull/195 for constant support.